use_case_examples/hybrid_model/infer_hybrid_llm_generate.py

```diff
@@ -0,0 +1,89 @@
+"""Showcase for the hybrid model converter."""
+import time
+from pathlib import Path
+
+import torch
+from torch.backends import mps
+from transformers import AutoModelForCausalLM, AutoTokenizer, PreTrainedTokenizer, TextStreamer
+
+from concrete.ml.torch.hybrid_model import FHEMode, HybridFHEModel
+
+if __name__ == "__main__":
+    configs = [
```

Comment:
can you explain what is in this config structure ?

Reply:
also, they are already defined in the compile file : maybe refactor this into a single config file ?

Reply:
I was thinking about it but creating a file just for that seemed a bit overkill.

Can do it if you think it's necessary.
I also thought about dumping a json in the compile file with the configuration that is then re-used in the inference script.
Would that work for you?

Reply:
I think having a config file is fine but as you wish, second solutions seems ok

in any case as Andrei said a comment explaining what these configs are would be great as well !

Reply:
Compilation dumps a json that can then be used by the client